lib/rules/disallow-newline-before-block-statements.js

```diff
@@ -199,6 +203,11 @@ module.exports.prototype = {
     }
 };
 
+function isBareBlock(node) {
+    return node.parentNode &&
+    node.parentNode.type === 'BlockStatement' || node.parentNode.type === 'Program';
```

Comment:
Yeah good point it would be `FunctionDeclaration` I guess I can add something like `|| node.parentNode.body && node.parentNode.body.type === 'BlockStatement';`?